activiti-services/activiti-services-query/activiti-services-query-rest/src/main/java/org/activiti/services/query/events/ProcessCompletedEvent.java

```diff
@@ -16,30 +16,22 @@
 
 package org.activiti.services.query.events;
 
-import org.activiti.services.query.model.ProcessInstance;
+import org.activiti.services.query.es.model.ProcessInstanceES;
```

Comment:
@gmalanga can you explain me why we need to use the *ES classes here? not sure to understand the logic of these changes here in this -rest module which should use the generic model.